packages/core/strapi/package.json

```diff
@@ -112,10 +112,6 @@
     "@strapi/logger": "4.15.5",
     "@strapi/pack-up": "4.15.5",
     "@strapi/permissions": "4.15.5",
-    "@strapi/plugin-content-manager": "4.15.5",
-    "@strapi/plugin-content-type-builder": "4.15.5",
-    "@strapi/plugin-email": "4.15.5",
-    "@strapi/plugin-upload": "4.15.5",
```

Comment:
I don't think these should be removed. They're imported as they're "core" plugins. https://github.com/strapi/strapi/blob/4ad8ac0235a95a1dfb3b9e2f33633b2fbd14ef71/packages/core/strapi/src/admin.ts

Reply:
maybe they were meant to be removed from the `core` package...?

Reply:
true they should be only removed from core/core idk why it's removed here 🤔